en/docs/guides/login/logging-into-simplesamlphp-using-is.md

```diff
@@ -0,0 +1,209 @@
+    ```
+
+2.  Install PHP and related extensions.
+
+    ``` java
+    # apt-get install php5  
+        
+    # apt-get install php5-cli  
+        
+    # apt-get install php5-common  
+        
+    # apt-get install php5-curl  
+        
+    # apt-get install php-pear  
+        
+    # apt-get install php5-mcrypt 
+    ```
+
+    !!! info 
+		For Ubuntu users, please install the following extension as well:
+		`            # apt-get install php5-json           `
+
+3.  Install SimpleSAMLphp using the following commands.
+
+    ``` java
+    # sudo mkdir /var/simplesamlphp/
+
+    # cd /var/simplesamlphp/  
+        
+    # wget https://github.com/simplesamlphp/simplesamlphp/releases/download/simplesamlphp-1.11.0/simplesamlphp-1.11.0.tar.gz  
+        
+    # tar xvf simplesamlphp-1.11.0.tar.gz  
+        
+    # mv simplesamlphp-1.11.0 simplesamlphp  
+        
+    # cd simplesamlphp  
+        
+    # cp -r metadata-templates/*.php metadata/  
+        
+    # cp -r config-templates/*.php config 
+    ```
+
+4.  Configure SimpleSAMLphp web in Apache.
+
+    ``` java
+    # cd /var/www/html
+        
+    # ln -s /var/simplesamlphp/simplesamlphp/www simplesaml 
+    ```
+
+5.  Start Apache.
+
+    ``` java
+    # apachectl start  
+    ```
+
+6.  Access the SimpleSAMLphp web app from the following location:
+    [http://localhost/simplesaml](http://localhost/simplesaml).
+7.  Set the SimpleSAMLphp administrator login configuration as follows:
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi config/config.php  
+    ```
+
+    1.  Search for ' **auth.adminpassword** ' and change its value
+        from the default and save the file.
+    2.  Click on ' **Login as administrator** ' from the web page
+        `                         http://localhost/simplesaml                       `
+        to test the configured value.
+
+8.  Add a service provider to SimpleSAMLphp.
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi config/authsources.php 
+    ```
+
+    1.  Add the following section to the file and save.
+
+        ``` java
+        'wso2-sp' => array(  
+            
+        'saml:SP',  
+            
+        // The entity ID of this SP.  
+            
+        // Can be NULL/unset, in which case an entity ID is generated based on the metadata URL.  
+            
+        'entityID' => 'simplesaml',  
+            
+        // The entity ID of the IdP this should SP should contact.  
+            
+        // Can be NULL/unset, in which case the user will be shown a list of available IdPs.  
+            
+        'idp' => 'https://localhost:9443/samlsso',  
+            
+        // The URL to the discovery service.  
+            
+        // Can be NULL/unset, in which case a builtin discovery service will be used.  
+            
+        'discoURL' => NULL,  
+            
+        ),
+        ```
+
+        Here we assume WSO2 IS is running in localhost on 9443.
```

Comment:
By default, WSO2 IS runs on localhost:9443. If you are using a different hostname or have configured a port offset, adjust the configurations above accordingly.